bigchaindb/core.py

```diff
@@ -274,8 +277,27 @@ def get_status(self, txid):
             or 'backlog'). If no transaction with that `txid` was found it
             returns `None`
         """
-        _, status = self.get_transaction(txid, include_status=True)
```

Comment:
It should be noted that `get_transaction()` uses `get_blocks_status_containing_tx()` and `query.get_transaction_from_backlog()` anyway, so this shouldn't be an impact on performance.